angio-frontend/src/modules/user/components/AccountActivationForm.vue

```diff
@@ -0,0 +1,174 @@
+                                name="password"
+                                prepend-inner-icon="lock"
+                                counter
+                                maxlength="120"
+                                outline
+                                required
+                        ></v-text-field>
+                        <v-expand-transition>
+                            <ul v-if="displayError" data-test-id="formMsg__list">
+                                <li class="red--text body-1">
+                                    {{ $t('user.component.activation.activationFormStep.error') }}
+                                </li>
+                            </ul>
+                        </v-expand-transition>
+                    </v-form>
+                </v-card-text>
+            </v-window-item>
+
+            <v-window-item v-bind:value="2">
+                <v-card-title class="headline font-weight-regular justify-space-between">
+                    <span>{{ $t('user.component.activation.activationSuccessStep.title') }}</span>
+                    <v-spacer></v-spacer>
+                </v-card-title>
+                <v-card-text class="subheading font-weight-light">
+                    <span>{{ $t('user.component.activation.activationSuccessStep.content') }}</span>
+                </v-card-text>
+            </v-window-item>
+        </v-window>
+
+        <v-divider></v-divider>
+
+        <v-card-actions>
+            <v-spacer></v-spacer>
+            <v-btn
+                    v-show="step === 1"
+                    v-bind:disabled="!valid"
+                    v-bind:loading="fetching"
+                    form="activation-form"
+                    type="submit"
+                    color="primary"
+                    flat
+                    round
+            >
+                {{ $t('user.component.activation.activationFormStep.button.send') }}
+            </v-btn>
+            <v-btn
+                    v-show="step === 2"
+                    to="/login"
+                    color="success"
+                    flat
+                    round
+            >
+                {{ $t('user.component.activation.activationSuccessStep.button.login') }}
+            </v-btn>
+        </v-card-actions>
+    </v-card>
+</template>
+
+<script lang="ts">
+    import {Component, Emit, Prop, Vue} from 'vue-property-decorator';
+    import BaseSingleFormContainer from '@/modules/common/components/BaseSingleFormContainer.vue';
+    import {UserActivationModel} from '@/modules/user/models/user';
+    import {CommonEvent} from '@/modules/common/helpers/commonEvent';
+
+    @Component({
+        components: {BaseSingleFormContainer}
+    })
+    export default class AccountActivationForm extends Vue {
+
+        @Prop()
+        public readonly fetching!: boolean;
+
+        @Prop()
+        public readonly step!: number;
+
+        @Prop()
+        public readonly displayError!: boolean;
```

Comment:
Pass error messages array from view component and show it if not empty.

Reply:
Done. Good idea. Apply it to LoginForm and ActivationForm.